File: Python/BMI.py
#BMI progamming

Height = int(input("Height >>"))
Width = int (input("Width >>"))

#公式 : 
#BMI = Width / (Height/100)**
# 體重過輕 BMI<18.5	 
# 健康體位 18.5<=BMI<24
# 體位異常 過重：24<=BMI<27
# 輕度肥胖：27 <= BMI < 30
# 中度肥胖：30 <= BMI < 35
# 重度肥胖：BMI >= 35

Height /=100 # 高度 換算 公尺
Height*=Height
BMI = Width/Height

print("%-5.2f" % BMI)
if(BMI < 18.5):
  print("體重過輕")
elif(BMI <24):
  print("健康體位")
elif(BMI <27):
  print("體位異常")
elif(BMI <30):
  print("輕度肥胖")
elif(BMI <35):
  print("中度肥胖")
else:
  print("重度肥胖")

File: C/GitBook_C/Chapter_4/Bmi.c
#include <stdio.h>// Input/Output 
#include <stdlib.h>
void BmiPint(float);

int main() //主
{

    float height ;
    int width ;
    float BMI;

//----------------INPUT-------------------//
    printf("enter height and width : ");
    scanf("%f %d",&height,&width);
//---------------OUTPUT-----------------//
    height = height/100; // height /=100;
    height=height*height;// 平方 height *=height;
    BMI =width/height;
    //if(BMI <= 18) printf("Thin\n");
    
    BmiPint(BMI);

    printf("BMI = %.2f \n",BMI);
    system("PAUSE");
    return 0;
}//main()

void BmiPint(float bmi)
{
    if(bmi <= 18){printf("Thin\n");}
    else if(bmi < 24) printf("normal\n");
    else if(bmi < 27) printf("overweight\n");
    else if(bmi < 30) printf("Mild obesity\n");
    else if(bmi < 35) printf("Moderately obese\n");
    else printf("Severe obesity\n");
}


//void 不回傳 
